perfzero/lib/perfzero/process_info_tracker.py

```diff
@@ -44,7 +44,8 @@ def __init__(self, output_dir):
 
   def start(self):
     self.start_time = time.time()
-    self.scheduler.enter(1, 1, self._update_process_info)  # pylint: disable=no-value-for-parameter
+    # 4th positional arg added to support Python2 for the short-term.
```

Comment:
Just for consistency, can we also add the 4th positional arg for `scheduler.enter` used in `perfzero/lib/perfzero/tensorflow_profiler.py`?

Reply:
Totally, I didn't know about that one.  I'll get it before merging this.  Thank you for pointing it out.

Reply:
Fixed